knowledge/example_test.go

```diff
@@ -125,34 +118,19 @@ func ExampleClient_GetDigitalTwinByIdentifier() {
 		_ = client.Close()
 	}()
 
-	response, err := client.GetDigitalTwinByIdentifier(context.Background(), &knowledge.Identifier{
-		ExternalID: "1234",
-		Type:       "Person",
-	})
-	if err != nil {
-		log.Fatalf("failed to invoke operation on IndyKite Client %v", err)
-	}
-	json := protojson.MarshalOptions{
-		Multiline: true,
+	returns := []*knowledgepb.Return{
+		{
+			Variable:   "n",
+			Properties: []string{},
```

Comment:
same here and other places, I won't mention all 😄

Reply:
updated